app/actions/navigation.js

```diff
@@ -78,3 +83,37 @@ export function resetToSelectServer(allowOtherServers) {
         });
     };
 }
+
+export function goToScreen(componentId, name, title, passProps = {}, options = {}) {
+    return (dispatch, getState) => {
+        const theme = getTheme(getState());
+        const defaultOptions = {
```

Comment:
I was thinking that we could look at `Navigation.setDefaultOptions` to do some of these for us, but I'm not sure how well that works when changing the theme.